src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -14,6 +15,7 @@
 
 from WMCore.Services.PhEDEx.PhEDEx import PhEDEx
 from WMCore.Lexicon import cmsname, slicedIterator
+pnn_regex = re.compile(r'^T[0-3%]((_[A-Z]{2}(_[A-Za-z0-9]+)*)?)')
```

Comment:
Sounds good. Maybe a comment there that the same string appears in Lexicon so we know to change one if we change the other? Also a comment on this. I guess T1_UK passes this? Maybe we have that for wild carding? Anyhow, as this re does not allow "." I think we are safe.